docs/rollup-api-json.js

```diff
@@ -172,4 +172,13 @@ const main = async () => {
         });
 };
 
-main();
+main().then(
+    () => {
+        console.log(colors.green('SUCCESS: API log files staged!'));
```

Comment:
Double quotes please. Makes me want prettier. 😝  (Or an easier way to apply ESLint to JS files outside the src dir throughout the repo.)